pyproject.toml

```
@@ -43,6 +43,7 @@ dependencies = [
     "pycollada<=0.9.1",
     "pycryptosat<=5.11.21",
     "python-sat[cryptosat]<=1.8.dev24",
+    "pymatching<=2.3.0",
```

```suggestion
    "pymatching<=2.3.1",
```

The latest version is v2.3.1 which fixed some bugs related to correlated matching.